recipes/k3d/yum_requirements.txt

```diff
@@ -0,0 +1,3 @@
+mesa-libGL-devel
+xorg-x11-server-Xvfb
+firefox
```

Comment:
Why is firefox needed?

Reply:
To get closely to the actually used CI setup.

Reply:
I don't understand. Can you please rephrase?

Reply:
Sorry. I meant to say that the actual CI setup for the package also installs firefox. Which If I understand correctly gets used in the grunt test task for javascript stuff.